src/rodal.js

```diff
@@ -14,7 +14,11 @@ const IS_IE_9 = UA && UA.indexOf('msie 9.0') > 0;
 const Dialog = props => {
     const animation = (props.animationType === 'enter' ? props.enterAnimation : props.leaveAnimation) || props.animation;
     const className = `rodal-dialog rodal-${animation}-${props.animationType}`;
-    const CloseButton = props.showCloseButton ? <span className="rodal-close" onClick={props.onClose} /> : null;
+    const CloseButton = props.showCloseButton ? <span className="rodal-close" onClick={props.onClose} onKeyPress={(event) => {
+        if (props.onCloseKeyPress) {
+            props.onCloseKeyPress(event);
+        }
```

Comment:
Thanks for PR! 

Maybe closing after keypress should be a default behavior, and it's unnecessary to add onCloseKeyPress prop

Reply:
Yes. Makes sense. 
Just made a change to handle it inside the Dialog itself.